glutton/conntrack.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
I would be grateful if you can help in understanding How I should organize my code? what should be in main and glutton package?